pkg/conduit/runtime.go

```diff
@@ -227,6 +239,9 @@ func (r *Runtime) Run(ctx context.Context) (err error) {
 		multierror.ForEach(err, func(err error) {
 			r.logger.Err(ctx, err).Msg("provisioning failed")
 		})
+		r.logger.Warn(ctx).
+			Msg("Conduit will be shut down due to a pipeline failure and strict mode enabled")
+		cancel()
```

Comment:
I think last time we spoke about this we didn't quite decide on the default mode, but it seems harsh to have Conduit fail because of provisioning as the default and only behavior. I'll bring it up at standup.